app/services/licences/fetch-licence-bills.service.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+/**
+ * Fetches all return logs for a licence which is needed for the view '/licences/{id}/returns` page
```

Comment:
I think the `/licences/{id}/returns` needs to be `/licences/{id}/bills` ?

Reply:
Done